controls/radgridview/export/excel/export-xlsx.md

```diff
@@ -30,15 +30,14 @@ The __ExportToXlsx__ method uses additional libraries so you need to add referen
 
 2. __ExportToXlsx(Stream stream, GridViewDocumentExportOptions options)__ - Expects the specified stream to which you are exporting data to and parameter of type GridViewDocumentExportOptions. The latter is used to set the following export options:
 
-- **Culture**: 
+- **Culture**: Sets a specific Culture.
 - **Items**: The collection of items to be exported.
 - **ShowColumnFooters**: Specifies whether column footers should be included on export.
 - **ShowGroupFooters**: Specifies whether group footers should be included on export.
-- **ShowGroupHeaders**: Specifies whether group headers should be included on export.
-- **ExportDefaultStyles**: Specifies whether the GridViewDataControl will be exported with its default styles.  
-- **ExcludedColumns**: Choose which columns to export. 
-- **ShowGroupRows**: Specifies whether the groups' items should be exported (this property was introduced in **R3 2019**).
-- **IgnoreCollapsedGroups**: Specifies whether the collapsed groups' items should be exported (this property was introduced in **R3 2019**).
+- **ShowColumnHeaders**: Specifies whether column headers should be included on export.
+- **ExportDefaultStyles**: Specifies whether the GridViewDataControl will be exported with its  
+- **ExcludedColumns**: Choose which columns to export.
+- **ShowGroupRows**: Specifies whether the groups' items should be included on export (this property was introduced in **R3 2019**).
```

Comment:
I think its better to say "Specifies whether the group rows should be included on export".